itest/supply_commit_test.go

```diff
@@ -1257,4 +1260,109 @@ func testSupplyVerifyPeerNode(t *harnessTest) {
 			peerFetchResp2.ChainData.SupplyRootHash,
 		),
 	)
+
+	// Step 8: Primary node mints another asset into the group and publishes
```

Comment:
Does this test fail without the current diff? 

Also we can move this into a new heler assertion, basically a sub-test, as this itest is already rather long.

Reply:
Also similar comment from the other PR: there're many duplicated assertions we can put into helper functions.

Reply:
That one is merged in now, so this can be rebased.